src/reports/package/package-report-footer.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedFC } from 'common/react/named-fc';
+import { serviceTitle, toolName } from 'content/strings/application';
+import { SectionProps } from 'reports/components/report-sections/report-section-factory';
+import { ToolLink } from 'reports/components/report-sections/tool-link';
+
+export type ReportFooterProps = Pick<SectionProps, 'environmentInfo'>;
+
+export const PackageReportFooter = NamedFC<ReportFooterProps>('ReportFooter', ({ environmentInfo }) => {
+    const { browserSpec, axeCoreVersion } = environmentInfo;
+    return (
+        <div className="report-footer-container">
+            <div className="report-footer" role="contentinfo">
+                This automated checks result was generated using the {serviceTitle}{' '}
```

Comment:
so `serviceTitle` resolves to `Accessibility Insights Service` (by looking at `src/content/strings/application.ts`) but the package can be run outside the service, right? If that's true then we are not fully solving this issue (of having AI-Web name in the report when it was not generated that way)

Reply:
can we make the title as an option that will be passed to this package?

Reply:
> Left some comments on specific files. I also notice on the screenshot we have **axe-core 3.3.2 and BROWSER_SPEC**... I'm guessing we are not passing browser spec values properly. I'm also wondering if we have missing unit tests around that.

That is the version of axe-core that created the test file and the magic string I passed into my example code that I generated the screenshot with.

Reply:
Added serviceName that can be passed in by the caller.